packages/contracts/contracts/TellerV2.sol

```diff
@@ -658,7 +662,8 @@ contract TellerV2 is
     function _repayLoan(
         uint256 _bidId,
         Payment memory _payment,
-        uint256 _owedAmount
+        uint256 _owedAmount,
+        bool _isLiquidation
```

Comment:
imo lets rename '_isLiquidation' to '_shouldWithdrawCollateral' because _repayLoan should not know what 'liquidation' is as a concept.  Separation of concerns in naming convention.